android/expoview/src/main/java/host/exp/exponent/notifications/managers/SchedulerManager.java

```diff
@@ -0,0 +1,150 @@
+package host.exp.exponent.notifications.managers;
+
+import android.content.Context;
+import com.raizlabs.android.dbflow.sql.language.Select;
+import org.unimodules.core.interfaces.Function;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import host.exp.exponent.notifications.exceptions.UnableToScheduleException;
+import host.exp.exponent.notifications.interfaces.SchedulerInterface;
+import host.exp.exponent.notifications.interfaces.SchedulersManagerInterface;
+import host.exp.exponent.notifications.schedulers.CalendarScheduler;
+import host.exp.exponent.notifications.schedulers.IntervalScheduler;
+
+class SchedulerManager implements SchedulersManagerInterface {
+
+  private boolean mFetchedFromDB = false;
```

Comment:
If every method starts with `fetchSchedulersMap()` you never act on a map as it is initialized, so instead of having a separate `mFetchedFromDB` variable you could discern between fetched/unfetched state with null-ness of `mSchedulersMap` null—unfetched, nonnull—fetched.